sidebars.js

```diff
@@ -56,6 +56,8 @@ const sidebars = {
       type: 'category',
       label: 'Capabilities',
       items: [
+        // zero
+        'docs/capabilities/custom-domains',
```

Comment:
Eventually we should probably put these all collected together with a prepended highlight like `ent` or something.